### DISCUSSION TITLE
不好意思，我該怎麼讓客戶端連線到伺服器？

### DISCUSSION THREAD

**Discussion by WhiteCloud0106** [Lost-MSth/Arcaea-server#111] [Q&A]
是需要一個網域或是改包嗎，我具體需要做甚麼
我在注意事項那邊有看到說處於同一個網路下
但我還是不太明白
能夠解釋一下就太謝謝了

**Comment by Lost-MSth** [comment]
其实在零零散散的 discussion issue 里面应该能看到相关线索

一切的核心是要让客户端去连接服务端（伺服器），方法有很多很多

v3.6.0 以上版本的客户端存在 ssl pin 和 ssl 证书校验，且 API 地址是加密的，一般来说需要干掉相关校验函数的调用，利用 DNS 劫持、流量转发（中间人攻击）来实现重定向链接
低版本怎么弄都行，API 地址没加密故直接改是最方便的

至于具体细节就请移步 google、**百度**、**某些博客**、github 上的**其它**相关仓库了
